requirements/base.in

```diff
@@ -12,8 +12,6 @@ cookies<2.3
 cryptography==3.2.1
 dictdiffer<0.9
 Django<2.3,>=2.2
-# using the latest django-angular (2.2 or later) requires a heavy rewrite of the registration form, so we use the old 1.0.0 version patched to work under Django 2
--e git+https://github.com/open-craft/django-angular.git@v1-with-django2-support#egg=django_angular
```

Comment:
Working on updating the `base.txt` and `dev.txt` files. Facing some issue with `pip-compile` command.

Reply:
Done!

Reply:
Done!